### DISCUSSION TITLE
Docker-Compose部署，404

### DISCUSSION THREAD

**Discussion by qlala1908** [metatube-community/jellyfin-plugin-metatube#427] [Q&A]
使用Docker-Compose部署时
执行
curl -sL https://raw.githubusercontent.com/metatube-community/metatube-sdk-go/main/docker-compose.yml
报错，404: Not Found

**Comment by Topbcy** [comment]
https://raw.githubusercontent.com/metatube-community/metatube-sdk-go/refs/heads/main/docker-compose.yaml

**Reply by qlala1908** [reply]
> https://raw.githubusercontent.com/metatube-community/metatube-sdk-go/refs/heads/main/docker-compose.yaml

yaml: line 216: mapping values are not allowed in this context

**Reply by Topbcy** [reply]
Try learn how to use docker.
